File: Payhub - SDK/c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/vt/TransactionTotals.cs
using PayHubSDK.com.payhub.ws.util;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;
using PayHubSDK.com.payhub.ws.api;

namespace PayHubSDK.com.payhub.ws.vt
{
    [DataContract]
    public class TransactionTotals
    {
        public static string TRN_TOTAL_LINK = "report/transactionTotals";

        [DataMember]
        public List<CreditCards> credit_cards;
        [DataMember]
        public AllTransactions all_transactions;       
        [DataMember]
        private List<Errors> errors;

        public List<Errors> Errors { get { return this.errors; } set { this.errors = value; } }
    }
}

File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/vt/TransactionTotals.java
package com.payhub.ws.vt;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.payhub.ws.api.Errors;

@JsonInclude(JsonInclude.Include.NON_NULL)  
public class TransactionTotals{
	public static final String TRN_TOTAL_LINK = "report/transactionTotals";

	private List<CreditCards> credit_cards;
	private AllTransactions all_transactions; 
	private List<Errors> errors;
	/**
	 * @return the credit_cards
	 */
	public List<CreditCards> getCredit_cards() {
		return credit_cards;
	}
	/**
	 * @param credit_cards the credit_cards to set
	 */
	public void setCredit_cards(List<CreditCards> credit_cards) {
		this.credit_cards = credit_cards;
	}
	/**
	 * @return the all_transactions
	 */
	public AllTransactions getAll_transactions() {
		return all_transactions;
	}
	/**
	 * @param all_transactions the all_transactions to set
	 */
	public void setAll_transactions(AllTransactions all_transactions) {
		this.all_transactions = all_transactions;
	}
	/**
	 * @return the errors
	 */
	public List<Errors> getErrors() {
		return errors;
	}
	/**
	 * @param errors the errors to set
	 */
	public void setErrors(List<Errors> errors) {
		this.errors = errors;
	}
	
	

}
